scripts/src/java/org/oppia/android/scripts/proto/file_content_validation_checks.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Checks to ensure the usage of any prohibited file content in the codebase
```

Comment:
Please add punctuation at the end of proto doc comments. Ditto elsewhere in the PR.

Reply:
done.